.azure-pipelines/azure-pipelines-linux.yml

```diff
@@ -1,36 +0,0 @@
-# This file was generated automatically from conda-smithy. To update this configuration,
```

Comment:
Are these yamls supposed to be deleted? (I'm not familiar with the conda-forge build system.)

Reply:
Ignore everything in conda-forge except `conda-forge.yml` and things in `recipes`. Everything else is auto-generated.